tools/src/keygen/Main.hs

```diff
@@ -106,22 +109,21 @@ getAvvmGenesis
     => AvvmStakeOptions -> m (GenesisCoreData, GenesisGtData)
 getAvvmGenesis AvvmStakeOptions {..} = do
     jsonfile <- liftIO $ BSL.readFile asoJsonPath
-    holder <- getHolderId asoHolderKeyfile
     case eitherDecode jsonfile of
         Left err       -> error $ toText err
         Right avvmData -> do
             avvmDataFiltered <- liftIO $ applyBlacklisted asoBlacklisted avvmData
             let totalAvvmStake = sum $ map aeCoin $ utxo avvmDataFiltered
             putText $ "Total avvm stake after applying blacklist: " <> show totalAvvmStake
-            pure $ genGenesis avvmDataFiltered asoIsRandcerts holder
+            -- warning: if testnet is not enabled,
+            -- gcdBootstrapAddresses will be empty. We should come up
+            -- with a way to pass real boot addresses to avvm
+            -- distribution.
+            pure $ genGenesis avvmDataFiltered asoIsRandcerts mempty
```

Comment:
It seems broken to me. AVVM part of utxo will have empty stake distribution. But that's not what we want. You should pass testnet stakeholders I suppose.